README.md

```diff
@@ -73,6 +73,12 @@ options for the offset in the file/text provided:
 ]
 ```
 
+To use the iOS SDK is to specified the `compilerargs` option. The value
```

Comment:
Correct .

If you do not put a `--` , it would have been interpreted as an option of SourceKitten.

For example , `-target` is would be treated as an option of sourcekitten. as a result, the unknown option error.

By the way , this feature is provided by the Commandant.